### DISCUSSION TITLE
Contributor avatars are missing when provided with a link of the username in release notes.

### DISCUSSION THREAD

**Discussion by Prathamesh010** [orhun/git-cliff#917] [Q&A]
[crawlee-python](https://github.com/apify/crawlee-python) uses git cliff to generate changelog and also release notes. But, Since last few releases, The releases have stopped showing the contributor avatars.

I went throught the [cliff.toml](https://github.com/apify/crawlee-python/blob/master/cliff.toml). The members are tags through urls and not directly, So, Github might not be able to parse it. But, This setup was working couple of release back and has started breaking now.

Is this a issue with github, git-cliff or a misconfiguration?

Script
https://github.com/apify/crawlee-python/blob/master/.github/workflows/run_release.yaml

@orhun Do you have any insights on this? 🙂

**Comment by orhun** [comment]
Thanks for the issue!

See https://github.com/apify/crawlee-python/pull/599
